api/src/schemas.ts

```diff
@@ -324,5 +321,47 @@ export const schemas = {
         })
       })
     }
+  },
+  // Challenges:
+  projectCompleted: {
+    body: Type.Object({
+      id: Type.String({ format: 'objectid', maxLength: 24, minLength: 24 }),
+      challengeType: Type.Optional(Type.Number()),
```

Comment:
It's optional in the current api:

```
  if ('challengeType' in req.body && !isNumeric(String(challengeType))) {
    log('challengeType', challengeType, isNumeric(challengeType));
    return res.status(403).json(isValidChallengeCompletionErrorMsg);
  }
```
 
Rather than checking the client, I figured we keep it optional for the MVP. If I add a TODO comment, is that okay?

Reply:
Sounds good - something for another PR